transmitter.js

```diff
@@ -2,15 +2,14 @@ let transmissionFailureCount = 0;
 
 function networkTransmitStub(celcius) {
     console.log(`Temperature to transmit: ${celcius} celcius`);
-
+    const roundedNumber = Math.round(Math.random());
+    // Map 0 to 200 and 1 to 500
+    const stubCode = (roundedNumber == 0) ? 200 : 500;

```

Comment:
If this generates all randoms above 0.5 in a test run, then the test will fail even if the code is ok.
Randomization of test data is a good technique, used in property-based testing. However, you need some predictability to control the test conditions. How would you bring that predictability here?